example/src/App.tsx

```
@@ -80,6 +81,58 @@ export default function App() {
+            Snackbar.show({
+              text: 'Verification dismissed',
+              duration: Snackbar.LENGTH_LONG,
+            });
+            break;
+          case ReclaimVerification.ExceptionType.SessionExpired:
+            Snackbar.show({
+              text: 'Verification session expired',
+              duration: Snackbar.LENGTH_LONG,
+            });
+            break;
+          case ReclaimVerification.ExceptionType.Failed:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)